targetingpacks.props

```diff
@@ -12,4 +13,25 @@
     <AssemblySearchPaths>$(AssemblySearchPaths);$(ContractOutputPath);{RawFileName}</AssemblySearchPaths>
   </PropertyGroup>
 
+  <!-- Adds test references to the targeting pack and runtime assemblies. -->
+  <Target Name="AddTestTargetingPackReferences" BeforeTargets="ResolveAssemblyReferences" Condition="'$(IsTestProject)' == 'true'">
+    <ItemGroup>
+      <!-- Whitelisted runtime assemblies that are OK to reference. -->
+      <RuntimeReferencedAssemblyNames Include="xunit.core" />
+      <RuntimeReferencedAssemblyNames Include="Xunit.NetCore.Extensions" />
+      <RuntimeReferencedAssemblyNames Include="xunit.assert" />
+      <RuntimeReferencedAssemblyNames Include="xunit.abstractions" />
+
+      <!-- Add Reference's to all files in the targeting pack folder, and to whitelisted items from the group above in the runtime folder. -->
+      <TargetingPackItems Include="%(TargetingPackDirs.Identity)/*" />
+      <Reference Include="%(TargetingPackItems.Filename)" Exclude="System.Private.CoreLib" /> <!-- TODO: System.Private.CoreLib shouldn't even be in the targeting pack. -->
```

Comment:
This is: https://github.com/dotnet/corefx/tree/dev/eng/src/Microsoft.TargetingPack.Private.CoreCLR/ref